tools/measurement/apksize/src/inappmessagingdisplay/java/com.google.apksize/InAppMessagingDisplay.java

```diff
@@ -15,24 +15,12 @@
 package com.google.apksize;
 
 import android.content.Context;
-import com.google.firebase.inappmessaging.FirebaseInAppMessagingClickListener;
-import com.google.firebase.inappmessaging.model.Action;
-import com.google.firebase.inappmessaging.model.CampaignMetadata;
-import com.google.firebase.inappmessaging.model.InAppMessage;
+import com.google.firebase.inappmessaging.FirebaseInAppMessaging;
 
 public class InAppMessagingDisplay implements SampleCode {
 
-  public class MyClickListener implements FirebaseInAppMessagingClickListener {
```

Comment:
Do you want to remove all of this? I think leaving this will make the test more equitable compared to the others.

Reply:
I've update the test code to be more comparable with the others. PTAL.